2/contrib/openshift/jenkins-version.txt

```diff
@@ -0,0 +1 @@
+2.319.3
```

Comment:
why dont we have same version as for 4.9, and >?
https://github.com/openshift/jenkins/blob/release-4.9/2/contrib/openshift/jenkins-version.txt

Reply:
I am planning to change jenkins version in other PR.
For this PR i have not changed jenkins version, just used what was there in 4.8 earlier https://github.com/openshift/jenkins/blob/release-4.8/2/Dockerfile.rhel8#L46